### PR TITLE
Add basic skeleton HTML

### PR DESCRIPTION
**Pull request by mad261**
It's a basic html to begin

### CODE REVIEW

**Comment by bassetone** [comment]
itako ity anao ity fa tokony hicommente an'le programme

**Comment by Pepe2022** [comment]
Ok; ajouter encore des contenues sur la page

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+
+    <title>Hello, world!</title>
+  </head>
+  <body>
+    <h1>Hello, world!</h1>
+
```
Comment (Pepe2022) [inline index.html:12]: mba asio code html oe eto jerena raha miova

Comment (bassetone) [reply index.html:12]: <b>ovaiko avy hatrany</b>

File: index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+
+    <title>Hello, world!</title>
+  </head>
+  <body>
+    <h1>Hello, world!</h1>
```
Comment (bassetone) [inline index.html:11]: <blink>asa ra m-blink</blink>

Comment (bassetone) [reply index.html:11]: tena miresadresak fotsiny ty

**Review by bassetone — COMMENTED** [review]
<h2>nakarina H2 -----Hello, world!</h2>

File: index.html (hunk 1 above)
Comment (bassetone) [reply index.html:12]: <h3><i>Hello, world!</i></h3>

**Pull request merged**